include/net/net_pkt.h

```diff
@@ -1705,6 +1705,19 @@ size_t net_pkt_available_payload_buffer(struct net_pkt *pkt,
  */
 void net_pkt_trim_buffer(struct net_pkt *pkt);
 
+/**
+ * @brief Remove @a length bytes from tail of packet
+ *
+ * @details This function does not take packet cursor into account. It is a
+ *          helper to remove unneeded bytes from tail of packet (like appended
+ *          CRC). It takes care of buffer deallocation if removed bytes span
+ *          whole buffer(s).
+ *
+ * @param pkt    Network packet
+ * @param length Number of bytes to be removed
+ */
```

Comment:
The return value should be documented.

Reply:
Good catch, thank! Documented now.

Reply:
While at it, I also added verification of return value in unit tests.